src/sagemaker_xgboost_container/mms_patch/mms_transformer.py

```diff
@@ -79,4 +80,4 @@ def transform(self, data, context):
                 logging.error(e)
                 return self.handle_error(context, e.status_code, e.message)
             else:
-                raise e
+                return self.handle_error(context, http.HTTPStatus.BAD_REQUEST, e.message)
```

Comment:
We are sending http status here, whereas we are relying on other status codes in line `81`, `51`, and `49`. Why is there a difference in status codes?

Secondly, for consistency, should we send back a 200OK as well for the good case?

Reply:
The other cases should already be propagated with a http status code from the exception. The 200 case is handled by MMS.